src/qibolab/__init__.py

```diff
@@ -1,16 +1,15 @@
-from .backends import MetaBackend, QibolabBackend, execute_qasm
-from .execution_parameters import AcquisitionType, AveragingMode, ExecutionParameters
-from .platform import Platform, create_platform
-from .version import __version__
+from . import backends, execution_parameters, platform, pulses, sequence, version
+from .backends import *
+from .execution_parameters import *
+from .platform import *
+from .pulses import *
+from .sequence import *
+from .version import *
```

Comment:
I think we need to expose a few more things here, for example the different channels, since they are needed for writing platforms. See for example here: https://github.com/qiboteam/qibolab_platforms_qrc/blob/367a26238ecb509d046d7d88c119b8f90e951c67/qw11qD/platform.py#L3

and potentially also `ConfigKinds`, which I believe is needed for registering instrument specific configs.

Of course, we also need the instruments and their special configs, but I am not sure if you are planning to expose them directly via `qibolab` or still keep the usual `from qibolab.instruments.qm import ...`.

Reply:
For sure, I'm planning to follow #790, but I didn't manage to finish yet. That's why it's still draft :)

Reply:
This is now done (in the many commits that followed)